6_visualize/src/create_gif_makefile.R

```diff
@@ -17,10 +17,11 @@ create_storm_gif_makefile <- function(makefile, task_plan, remake_file) {
     packages=c('dplyr', 'scipiper'),
     sources=c(
       '6_visualize/src/create_animation_frame.R',
-      '6_visualize/src/prep_datetime_fun.R',
       '6_visualize/src/prep_gage_sites_fun.R',
       '6_visualize/src/prep_storm_point_fun.R',
       '6_visualize/src/prep_precip_fun.R',
-      '6_visualize/src/prep_spark_line_fun.R'),
+      '6_visualize/src/prep_spark_line_fun.R',
```

Comment:
one thing we learned in pipelines III is that we've got some reproducibility issues with running the task tables unless we use `sources` as a depends or as inputs to the task table combiner function/output. 

We should consider if we want to make minimal edits to how this pipeline works for pragmatic reasons, or update it with the preferred task table structure.